literature.bib

```diff
@@ -17570,6 +18056,49 @@ @article{M92
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 1991 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{ISY:DAC91,
+	title = {{Minimization of Binary Decision Diagrams Based on Exchanges of Variables}},
+	author = {Ishiura, Nagisa and Sawada, Hiroshi and Yajima, Shuzo},
+	booktitle = DAC,
+	pages = {472--473},
+	publisher = ACM,
+	address = NY,
+	year=1991
+}
+
+
+@inproceedings{FMK:EURODAC91,
+	title = {{On Variable Ordering of Binary Decision Diagrams for the Application of Multi-Level Logic Synthesis}},
+	author = {Fujita, Masahiro and Matsunaga, Yusuke and Kakuda, Taeko},
+	booktitle = EURODAC,
+	pages = {50--54},
+	publisher = IEEE,
+	address = Washington,
+	year = 1991,
+}
+
+@inproceedings{BRKM:DAC91,
+	title={{Heuristics to Compute Variable Orderings for Efficient Manipulation of Ordered Binary Decision Diagrams}},
+	author={Butler, Kenneth M and Ross, Don E and Kapur, Rohit and Mercer, M Ray},
```

Comment:
Put dots after abbreviated names?
`Butler, Kenneth M. and Ross, Don E. and Kapur, Rohit and Mercer, M. Ray`

Reply:
Not according to DBLP or the BitTeX docs.